pesto-cli/pesto/cli/core/docker_builder.py

```diff
@@ -57,7 +57,7 @@ def _env_variables(self):
         return {
             'PESTO_PROFILE': self.build_config.full_version,
             **self.environments,
-            'PYTHONPATH': self._python_path
+            'PYTHONPATH': ":".join(["$PYTHONPATH", self._python_path])
```

Comment:
Is it still safe when no PYTHONPATH is present in the environment? We will get a PYTHONPATH starting with `:`. Do you see a way to check if the PYTHONPATH is set in the base image?

Reply:
Probably not, I will find an other way to do that.

Reply:
I change the way to set the PYTHONPATH. Can you check the new version ? I use [parameter expansion](https://wiki.bash-hackers.org/syntax/pe#use_an_alternate_value) to make it work.

Reply:
I tested your branch, but I still get `ENV PYTHONPATH=${PYTHONPATH:-/opt/ws-l1c-to-3d}` in the output dockerfile...

In the container: `PYTHONPATH -> /usr/local/otb/lib/python::/opt/ws-l1c-to-3d `

Reply:
You could use a one-liner:  `$PYTHONPATH${PYTHONPATH:+:}/opt/service`

Reply:
changes done here : https://github.com/CS-SI/pesto/pull/1/commits/1fcf15050862c38303c84ee5501cdabdfac71594